spec/support/data/renewal_raw_json.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
Sorry, but this can be deleted now right?

Reply:
sure, I also removed it from the other raw json files.